modules/tinymce/src/plugins/wordcount/main/ts/core/GetText.ts

```diff
@@ -5,25 +5,26 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { CharacterData, Node } from '@ephox/dom-globals';
+import { Node, Text } from '@ephox/dom-globals';
+import { Unicode } from '@ephox/katamari';
 import TreeWalker from 'tinymce/core/api/dom/TreeWalker';
 import Schema, { SchemaMap } from 'tinymce/core/api/html/Schema';
 
+const trimZwsp = (text: string) => text.replace(new RegExp(Unicode.zeroWidth, 'g'), '');
```

Comment:
🤔
https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/text/Zwsp.ts#L24

Reply:
Can't use that! It's in the internal API, but yes that's where I copied it from.

Reply:
I realise you can't access it at runtime, and this isn't a common thing in our plugins, but could you link to it and let the rollup copy it in for you? 🤔

Reply:
Rollup is configured to block anything outside the `api` module directory, to prevent it accidentally being used. So that fails as well unfortunately.

Reply:
https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/Gruntfile.js#L102 for reference.

Reply:
right, as I was writing it I figured one of us might have made it impossible to pull internal dependencies by accident

Maybe move `zwsp` to katamari? 😂

Reply:
So you can use types from anywhere, but actual code can only be used from the `api` directory.

Reply:
Dang GitHub not showing updated comments again :frowning: Yeah, that's feasible, so will look into that.